Makefile

```diff
@@ -30,8 +30,8 @@ all: build unit-test system-test ubuntu-tests
 # 'all-CI' target is used by the scripts/CI.sh that passes appropriate set of
 # ENV variables (from the jenkins job) to run OS (centos, ubuntu etc) and
 # sandbox specific(vagrant, docker-in-docker)
-all-CI: stop clean start
-	make ssh-build
+all-CI: stop clean
+	make checks run-build start ssh-build
```

Comment:
This is already necessary now. The system tests are compiled on the host anyway.

The code is compiled twice now. The first build is done on the host to ensure that the code can be compiled. The second build is done to ensure that we run binaries netplugin, netmaster and netctl binaries built with the Go version specified in our Makefile.